app/services/flow/flow_state_machine.rb

```diff
@@ -3,7 +3,7 @@ module FlowStateMachine
     extend ActiveSupport::Concern
 
     included do
-      before_action :fsm_initialize
+      before_action :flow_state_machine_initialize
```

Comment:
If we're renaming it, I might propose to move the verb to the beginning for consistency with other `before_action` and to make it readable as an expression "before action, initialize flow state machine".

```suggestion
      before_action :initialize_flow_state_machine
```

Reply:
@aduth very good suggestion, thank you.

Reply:
Done.